contrib/test_jenkins.sh

```diff
@@ -180,16 +180,31 @@ prepare() {
 # Build documentation
 #
 build_docs() {
-	echo " ==== Build docs only ===="
+	doxy_ready=0
+	doxy_target_version="1.8.11"
+	doxy_version="$(doxygen --version)"
+
 	# Try load newer doxygen if native is older than 1.8.11
-	if ! (echo "1.8.11"; doxygen --version) | sort -CV
+	if ! (echo $doxy_target_version; echo $doxy_version) | sort -CV
+	then
+		if module_load tools/doxygen-1.8.11 && doxygen --version
```

Comment:
why need to run "doxygen --version"? module_load should return nonzero if it fails..

Reply:
yes, you're right
btw, `module_avail` returns 0 even if `module` doesn't exist itself - fixed this in separate commit (now `return 1` if it's a case)